.travis.yml

```diff
@@ -1,6 +1,5 @@
 language: node_js
 node_js:
   - "node"
-  - "12"
-  - "10.12.0"
+  - "15.14.0"
```

Comment:
Removing v10 make sense since it is not supported. But we should be testing all supported Node.js versions: 12, 14, 16, and 17. https://nodejs.org/en/about/releases/

Reply:
Simply forgot this there, sorry.